### PR TITLE
feat(docs): add support documentation

### PR DESCRIPTION
**Pull request by dakahn**
I wrote some support and maintenance documentation based on my experience on that project over the previous block of work. 🏄 

Please feel free to jump in and suggest corrections or clarifications 👍

### CODE REVIEW

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @dakahn for writing this up!

**Review by dakahn — COMMENTED** [review]
- [ ] need to add mention of adding an issue to appropriate project boards

File: docs/guides/support.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+- Incoming pull requests (code review, initial communication, ongoing
+  discussion, rejection or merge) from the community (i.e. not from the core
+  team)
+- Triage and shepherd issues to close. Answer questions and help our users find
+  solutions to their problems
+- Does not include feature enhancements or new feature requests, which currently
+  fall under https://github.com/orgs/carbon-design-system/projects/8 (as of
+  4/30/19)
+
+### What repositories do we support?
```
Comment (joshblack) [inline docs/guides/support.md:19]: This section might need to be updated with the monorepo switch, right?

File: docs/guides/support.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+## Issues
+
+When our users want to submit a new issues to one of our repositories they're
+presented with the option to select from one of four templates. These templates
+ask a series of specific questions that make first-aid and triage go smoothly.
+They help us help them.
+
+_If the user hasn't taken the time to fill out one of our templates feel free to
+ask them to do so. Typically you'll need at least a good description of the
+problem, a walkthru of how to recreate the problem and a minimal test case to
```
Comment (joshblack) [inline docs/guides/support.md]:
```suggestion
problem, a walkthrough of how to recreate the problem and a minimal test case to
```

**Review by joshblack — APPROVED** [review]

**Pull request merged**